default_flashing_period: 0.25
default_alternate_period: 1
default_chase_period: 4
default_colorwipe_period: 5
default_rainbow_period: 5
default_rainbowcycle_period: 5
default_rainbowchase_period: 5
default_goup_period: 5
default_goupanddown_period: 5
default_breath_period: 4
default_snake_period: 1.5

led_offset: 8 #offset entre la led1 et la led la plus à l'arrière du robot afin de centrer les annimations
enable_led_ring_bcm_pin: 27

use_mesh: False
simulation_led_mesh_path: "package://niryo_robot_description/meshes/led_ring/led.dae"

simulation_led_ring_markers_publish_rate: 20
led_ring_markers_publish_rate: 5
